src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -314,10 +314,53 @@ public void testBanditColdStartAction() {
     BanditLogData capturedBanditLog = banditLogCaptor.getValue();
     assertEquals("cold-start-bandit-experiment-bandit", capturedBanditLog.experiment);
     assertEquals("cold-start-bandit", capturedBanditLog.variation);
-    assertEquals("subject2", capturedBanditLog.subject);
+    assertEquals("subject1", capturedBanditLog.subject);
+    assertEquals(Map.of(), capturedBanditLog.subjectNumericAttributes);
+    assertEquals(Map.of(), capturedBanditLog.subjectCategoricalAttributes);
+    assertEquals("option1", capturedBanditLog.action);
+    assertEquals(Map.of(), capturedBanditLog.actionNumericAttributes);
+    assertEquals(Map.of(), capturedBanditLog.actionCategoricalAttributes);
+    assertEquals(0.3333, capturedBanditLog.actionProbability, 0.0002);
+    assertEquals("falcon cold start", capturedBanditLog.modelVersion);
```

Comment:
☝️ key part of this test: action was selected with 1/3 probability and logged as a cold start